--- rc-1.7.1/acinclude.m4.orig	2002-11-27 17:28:15.000000000 +0100
+++ rc-1.7.1/acinclude.m4	2007-06-02 23:05:43.277290269 +0200
@@ -197,6 +197,8 @@
 
 dnl Where is tgetent()?
 AC_DEFUN(RC_LIB_TGETENT, [
+    AC_CHECK_LIB(tinfo, tgetent,
+	rc_lib_tgetent=-ltinfo,
 	AC_CHECK_LIB(termcap, tgetent,
 		rc_lib_tgetent=-ltermcap,
 		AC_CHECK_LIB(ncurses, tgetent,
@@ -204,4 +206,5 @@
 			AC_MSG_ERROR(tgetent not found)
 		)
 	)
+    )
 ])
